controllers/usersController.js

```diff
@@ -92,21 +93,30 @@ const addNewUser = async (req, res) => {
 }
 
 /**
- * Update user
+ * Update the user
  *
  * @param req {Object} - Express request object
- * @param req.params.id {string} - User id
  * @param req.body {Object} - User object
  * @param res {Object} - Express response object
  */
-const updateUser = async (req, res) => {
+const updateSelf = async (req, res) => {
   try {
-    const user = await userQuery.addOrUpdate(req.body, req.params.id)
+    const token = (req.headers.authorization.split(' '))[1]
```

Comment:
I thought we take our token from the `rds-session` cookie. What is this authorization header?

Reply:
Thats `Bearer JWT` token

Reply:
What is that @whyDontI?

Reply:
Sorry, I totally forgot that we use Cookie in prod.
Well have to change this code to get token from cookie.
My bad 😶